spec/music_album_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './music/music_album'
+
+describe 'MusicAlbum' do
+  before(:each) do
+    @album = MusicAlbum.new(on_spotify: true, can_be_archived: true)
+  end
+
```

Comment:
Instead of using before(:each), the suggestion is to use let(:album) { MusicAlbum.new(on_spotify: true, can_be_archived: true) } to define the album object. This will make the code more concise and improve readability.